hudi-common/pom.xml

```diff
@@ -204,5 +204,18 @@
       <scope>test</scope>
     </dependency>
 
+
+    <!-- alluxio -->
```

Comment:
this seems to be done, just to get the class name programmatically?  Can we not take this dependency? (we have kept it that way for all the `fs` impl so far, if we want to change it, then need to change/add testing etc uniformly, which is a separate body of work/jira IMHO)>